aioamqp/__init__.py

```diff
@@ -3,6 +3,7 @@
 import sys
 import ssl as ssl_module  # import as to enable argument named ssl in connect
 from urllib.parse import urlparse
+import os
```

Comment:
It looks like imports are mostly sorted here so keeping them this way would be nice. Tools such as `isort` can help you deal with that.